### PR TITLE
#4913 Align tab bar icons with design

### PR DESCRIPTION
**Pull request by ozanmanav-ng**

### CODE REVIEW

File: packages/wallet/app/router/AppAuthRouterTabs.tsx [hunk 1]
```diff
@@ -52,14 +54,25 @@ const tabConfig = [
...
   const { navigationTheme } = useTheme();
+  const insets = useSafeAreaInsets();
+  const tabHeight =
+    insets.bottom === 0 ? BASE_TAB_HEIGHT : BASE_TAB_HEIGHT + insets.bottom - TAB_PADDING;
 
   return (
     <Tab.Navigator
       initialRouteName={EAppRoutes.home}
       tabBarOptions={{
         inactiveTintColor: navigationTheme.colors.text,
```
Comment (desfero) [inline packages/wallet/app/router/AppAuthRouterTabs.tsx:70]: inactive tint color should be a base gray one. We should also specify active tint color to base gray (by default it's base gray but we never know when react-navigation updates styling)

File: packages/wallet/app/router/AppAuthRouterTabs.tsx [hunk 2]
```diff
@@ -52,14 +54,25 @@ const tabConfig = [
...
+        style: {
+          paddingBottom: insets.bottom + TAB_PADDING,
```
Comment (desfero) [inline packages/wallet/app/router/AppAuthRouterTabs.tsx:72]: what that change is fixing exactly? I'm trying to spot the difference but it seems the same

File: packages/wallet/app/router/AppAuthRouterTabs.tsx [hunk 3]
```diff
@@ -68,7 +81,9 @@ const AppAuthRouterTabs: React.FunctionComponent = () => {
           name={name}
           component={component}
           options={{
-            tabBarLabel: ({ color }: TBarProps) => <MenuLabel style={{ color }}>{name}</MenuLabel>,
+            tabBarLabel: ({ color }: TBarProps) => (
+              <MenuLabel style={[styles.label, { color }]}>{name}</MenuLabel>
```
Comment (desfero) [inline packages/wallet/app/router/AppAuthRouterTabs.tsx:85]: Let's use `MenuLabelBold`

File: packages/wallet/app/router/AppAuthRouterTabs.tsx [hunk 4]
```diff
@@ -81,4 +96,11 @@ const AppAuthRouterTabs: React.FunctionComponent = () => {
   );
 };
 
+const styles = StyleSheet.create({
+  label: {
+    fontWeight: "500",
+    letterSpacing: 0.1,
```
Comment (desfero) [inline packages/wallet/app/router/AppAuthRouterTabs.tsx:102]: btw do we need that later spacing? Why we don't use the one that was set by default in typography? If there is some issue with dithering we can try to reduce `letterSpacing to 0.1` but that should be also done on a typography level.

Also we don't need `fontWeight` probably if we use `menuLabelBold`

**Review by desfero — CHANGES_REQUESTED** [review]

File: packages/wallet/app/router/AppAuthRouterTabs.tsx (hunk 1 above)
Comment (ozanmanav-ng) [reply packages/wallet/app/router/AppAuthRouterTabs.tsx:70]: Active tint color looks the same, according to Design. 

image

https://app.zeplin.io/project/5e6f48a43076c61012fd3793/screen/606c646c2493089832b1e969

File: packages/wallet/app/router/AppAuthRouterTabs.tsx (hunk 2 above)
Comment (ozanmanav-ng) [reply packages/wallet/app/router/AppAuthRouterTabs.tsx:72]: You should check with iphone 11 and iphone 7. I used it to bring the text and icon closer together.

File: packages/wallet/app/router/AppAuthRouterTabs.tsx (hunk 3 above)
Comment (ozanmanav-ng) [reply packages/wallet/app/router/AppAuthRouterTabs.tsx:85]: done

File: packages/wallet/app/router/AppAuthRouterTabs.tsx [hunk 5]
```diff
@@ -52,14 +54,25 @@ const tabConfig = [
...
 
   return (
     <Tab.Navigator
       initialRouteName={EAppRoutes.home}
       tabBarOptions={{
         inactiveTintColor: navigationTheme.colors.text,
+        style: {
+          paddingBottom: insets.bottom + TAB_PADDING,
+          paddingTop: TAB_PADDING,
+          height: tabHeight,
```
[the remark this answers is not in the document]
Comment (ozanmanav-ng) [reply packages/wallet/app/router/AppAuthRouterTabs.tsx:74]: When I talked to Andrew,  he asked me to use 56px on non-insets devices. That's why we have to use the height. I have refactored as follows. I used the padding to bring the icon and text closer together.
 
image

File: packages/wallet/app/router/AppAuthRouterTabs.tsx (hunk 2 above)
Comment (ozanmanav-ng) [reply packages/wallet/app/router/AppAuthRouterTabs.tsx]: I refactored to this:
image

File: packages/wallet/app/router/AppAuthRouterTabs.tsx (hunk 1 above)
Comment (ozanmanav-ng) [reply packages/wallet/app/router/AppAuthRouterTabs.tsx:67]: done

**Review by desfero — APPROVED** [review]

**Review by KonradFilipiak — APPROVED** [review]

**Pull request merged**